Sources/MBNetworking/Networkable+PinningDelegate.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  Networkable+PinningDelegate.swift
+//  Networking
+//
+//  Created by Rasid Ramazanov on 29.01.2020.
+//  Copyright © 2020 Mobven. All rights reserved.
+//
+
+import Foundation
+import Security
+
+protocol URLSessionPinningDelegateProtocol: URLSessionDelegate {
+    var certificatePaths: [String] { get set }
+}
+
+func createURLSessionPinningDelegate() -> URLSessionPinningDelegateProtocol {
```

Comment:
URLSessionPinningDelegateProtocol içinde static tanımlayıp kullanabiliriz.

Reply:
Static member 'create' cannot be used on protocol metatype '(any URLSessionPinningDelegateProtocol).Type'

Bundan dolayi kullanamiyorum.

Reply:
URLSessionPinningComposer gibi bir enum oluşturup, onun içinde static çağırsak?

Reply:
Duzenledim